bitrise.yml

```diff
@@ -1074,8 +1075,18 @@ workflows:
                 # debug log
                 set -x
 
+                echo "Listing files in /sdcard/Pictures/UiTestScreenShots/"
+                adb shell ls /sdcard/Pictures/UiTestScreenShots/ || true
+
+                echo
+
+                echo "Listing files in /sdcard/Download/UiTestHierarchy/"
```

Comment:
nit: is listing the best word for this? Or should there be another echo comment on L1085 to let people know the next start of pulling the data is happening too?

Reply:
Yes, it would be better to add an another echo, good catch 👍